frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -27,6 +27,11 @@ export const listPage = {
     numberOfActiveFiltersShouldBe: (numFilters: number) => {
       cy.get("[class='pf-c-toolbar__item pf-m-chip-group']").should('have.length', numFilters);
     },
+    clickSearchByDropdown: () => {
+      cy.get('.pf-c-toolbar__content-section').within(() => {
```

Comment:
I'm guessing since this is a PF component we can't add a better selector.

Reply:
yes, exactly